src/components/dialogs/script-dialog.js

```diff
@@ -107,6 +107,7 @@ const ScriptDialog = ({ id, open, onClose, onError, title, type }) => {
                 description: description,
                 script: aceEditorContent,
                 type: filterSubtype.SCRIPT,
+                subtype: filterSubtype.SCRIPT,
```

Comment:
are both of these used ? It is weird that type and subtype use the same filterSubType enum. Should probably be contingencyListSubtype.SCRIPT no ?

Reply:
Type was wrong yes. I did the correction

Reply:
Actually, type is used for script filter edition. I kept 
`type: FilterType.SCRIPT,`
It's clean and it works now